rllib/algorithms/ppo/tests/test_ppo_with_rl_module.py

```diff
@@ -85,7 +85,7 @@ def test_ppo_compilation_and_schedule_mixins(self):
                 enable_connectors=True,
             )
             .callbacks(MyCallbacks)
-            .experimental(_enable_rl_module_api=True)
+            .rl_module()
```

Comment:
Yeah, this is confusing syntax. How would the user know, that calling this method without arguments automatically sets `_enable_rl_module_api=True`. Maybe explicitly pass in the `_enable_rl_module_api=True` for now?

Reply:
After internal discussion we decided to make it explicit requirement for the user to enable the flag. If for any reason `rl_module()` api is used but user have not explicitly turned it on, we throw a gentle warning saying hey you may have forgotten to enable this.